app/datastructure/ds_methods.py

```diff
@@ -0,0 +1,125 @@
+from datetime import datetime as dt
+# from werkzeug.security import generate_hash_password, check_password_hash
+
+
+users = []
```

Comment:
Good work here two, However it would be nice if we use class variables instead of global variables.  read about class variables and make the necessary changes.